docs/docs/migration_notes.md

```diff
@@ -5,6 +5,54 @@ keywords: [sandbox, aztec, notes, migration, updating, upgrading]
 ---
 
 Aztec is in full-speed development. Literally every version breaks compatibility with the previous ones. This page attempts to target errors and difficulties you might encounter when upgrading, and how to resolve them.
+## 0.67.2 
+### [archiver, node, pxe] Remove contract artifacts in node and archiver and store function names instead 
+Contract artifacts were only in the archiver for debugging purposes. Instead function names are now (optionally) emitted
+when registering contract classes
+
+Function changes in the Node interface and Contract Data source interface:
+```diff
+- addContractArtifact(address: AztecAddress, artifact: ContractArtifact): Promise<void>;
++ registerContractFunctionNames(address: AztecAddress, names: Record<string, string>): Promise<void>;
+```
+
+So now the PXE registers this when calling `registerContract()`
+```
+await this.node.registerContractFunctionNames(instance.address, functionNames);
+```
+
+Function changes in the Archiver
+```diff
+- addContractArtifact(address: AztecAddress, artifact: ContractArtifact)
+-  getContractArtifact(address: AztecAddress)
++  registerContractFunctionNames(address: AztecAddress, names: Record<string, string>): Promise<void>
+```
+
+### [fees, fpc] Changes in setting up FPC as fee payer on AztecJS and method names in FPC
+On AztecJS, setting up `PrivateFeePaymentMethod` and `PublicFeePaymentMethod` are now the same. The don't need to specify a sequencer address or which coin to pay in. The coins are set up in the FPC contract!
+
+```diff
```

Comment:
based on [our slack convo](https://aztecprotocol.slack.com/archives/C04PUD9AA4W/p1734598764681099?thread_ts=1734598584.195409&cid=C04PUD9AA4W), @benesjan I think we should revert this